stixpy/map/stix.py

```diff
@@ -0,0 +1,79 @@
+import astropy.units as u
+from sunpy.map import GenericMap
+
+__all__ = ['STIXMap']
+
+
+class STIXMap(GenericMap):
+    r"""
+    A class to represent a STIX Map.
+    """
+
+    @classmethod
+    def is_datasource_for(cls, data, header, **kwargs):
+        return str(header.get('TELESCOP', '')).endswith('STIX')
+
+    # @property
+    # def coordinate_system(self):
+    #     """
+    #     Coordinate system used
+    #
+    #     Overrides the values in the header which are not understood by Astropy WCS
+    #     """
+    #     return SpatialPair("SXLN-TAN", "SXLT-TAN")
+    #
+    # @property
+    # def spatial_units(self):
+    #     return SpatialPair(u.arcsec, u.arcsec)
+
+    def plot(self, **kwargs):
```

Comment:
FYI for future NDCube-backed Maps, this will have to be moved into a `STIXPlotter` object which is attached to the `STIXMap`.  It won't change the user-facing API though.  However, no action needed on this for now.